plugins/modules/aci_cloud_subnet.py

```diff
@@ -54,17 +56,24 @@
     - Address of cloud cidr.
     type: str
     required: true
-  availability_zone:
+  aws_availability_zone:
```

Comment:
Why change this input variable instead of just adding to alias?

Reply:
The reason is to make it more clearer and explicit.